syntax = "proto3";

service Simulator {
   rpc RunSimulation(Simulation) returns (SimResponse) {}
}

message Simulation {
    int32 numACells = 1;
    int32 numBCells = 2;
    double gammaA = 3;
    double gammaB = 4;
    int32 xValue = 5;
    int32 yValue = 6;
    int32 zValue = 7;
    int32 stopTime = 8;
}

message SimResponse{
    string message = 1;
}